src/patternfly/components/Masthead/masthead.scss

```diff
@@ -3,174 +3,108 @@ $pf-v5-c-masthead--breakpoint-map: build-breakpoint-map();
 $pf-v5-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
 :root {
-  --#{$masthead}--PaddingLeft: var(--#{$masthead}--inset);
-  --#{$masthead}--PaddingRight: var(--#{$masthead}--inset);
+  --#{$masthead}--inset: var(--pf-t--global--spacer--lg);
+  --#{$masthead}--PaddingBlockStart: var(--pf-t--global--spacer--lg);
```

Comment:
I think this should be a md spacer - the original sketch designs are slightly different from the figma components, but design verified that we should go by the Figma component if there's a difference.